modules/javafx.controls/src/main/java/javafx/scene/control/Tooltip.java

```diff
@@ -1028,6 +1031,11 @@ public boolean isSettable(Tooltip.CSSBridge cssBridge) {
          * stop the HIDE_TIMER, hide the tooltip, and start the LEFT_TIMER.
          */
         private EventHandler<MouseEvent> LEAVING_HANDLER = (MouseEvent event) -> {
+            if( mouseInsideTooltip )
```

Comment:
Can you reformat this to follow our code style guidelines? Add a space after the `if`, remove the spaces after `(` and before `)`, and move the opening `{` to the same line as the `if`.